src/main/java/org/jenkinsci/plugins/configfiles/builder/ConfigFileBuildStep.java

```diff
@@ -47,21 +46,26 @@ public ConfigFileBuildStep(List<ManagedFile> managedFiles) {
     public List<ManagedFile> getManagedFiles() {
         return managedFiles;
     }    
-
-    @Override
-    public boolean perform(AbstractBuild<?, ?> build, Launcher launcher, BuildListener listener) throws IOException, InterruptedException {
-
-        if (build.getWorkspace() == null) {
+	
+	@DataBoundSetter
+	public void setManagedFiles(List<ManagedFile> managedFiles) {
```

Comment:
You do not need `@DataBoundSetter` if the property is already bound in a `@DataBoundConstructor`.